pkg/controllers/clusterset/clusterrole/rbac.go

```diff
@@ -36,6 +37,10 @@ func buildAdminRoleRules(clustersetName string) []rbacv1.PolicyRule {
 			Groups(managedclusterGroup).
 			Resources("managedclusters").
 			RuleOrDie(),
+		clusterrbac.NewRule("update").
+			Groups(registerGroup).
+			Resources("managedclusters/accept").
```

Comment:
this seems suspicious. I do not think admin for a clusterset can accept ANY cluster join

Reply:
@ldpliu please add a todo here so we handle  this issue in another pr.